src/Tickets/Admin/Upsell.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace TEC\Tickets\Admin;
+
+use \Tribe\Admin\Upsell_Notice;
+
+/**
+ * Class Upsell
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Admin
+ */
+class Upsell {
+	
+	/**
+	 * Method to register Upsell-related hooks.
+	 * 
+	 * @since TBD
+	 */
+	public function hooks() {
+		add_action( 'tribe_events_tickets_pre_edit', [ $this, 'maybe_show_capacity_arf' ] );
+	}
+	
+	/**
+	 * Maybe show upsell for Capacity and ARF features.
+	 * 
+	 * @since TBD
+	 */
+	public function maybe_show_capacity_arf() {
+		// If they already have ET+ activated, then bail.
+		if ( class_exists( 'Tribe__Tickets_Plus__Main' ) ) {
+			return;
+		}
+
+		tribe( Upsell_Notice\Main::class )->render( [
+			'classes' => [
+				'tec-admin__upsell-event_tickets-capacity_arf'
+			],
+			'text'    => esc_html__( 'Get individual information collection from each attendee and advanced capacity options with' , 'event-tickets' ),
```

Comment:
Same comment as https://github.com/the-events-calendar/event-tickets/pull/2390#discussion_r858595912

Reply:
I also left a reply in #2390